### DISCUSSION TITLE
Interacting with binance smart chain contract method

### DISCUSSION THREAD

**Discussion by RajKumar7777** [trustwallet/wallet-core#1587] [Q&A]
Hi,

   I need to interact with binance smart chain and bep20 contract method.

   Contract Address: 0xc6E070BCb7823E483164bB41D2297DCd8b1FAb25
   Contract Abi: https://api-testnet.bscscan.com/api?module=contract&action=getabi&address=0xc6E070BCb7823E483164bB41D2297DCd8b1FAb25&apikey=

   I searched the wallet core tests..I could not able to found the solution

   How to interact with these contract method..Please help to sort out this.

**Comment by optout21** [comment]
Check this test for building a function call: `EncodeFuncCase2`
Check this test for putting together and signing a transaction for a contract call:  `SignERC20TransferAsGenericContract`
Please do your homework.

**Comment by RajKumar7777** [comment]
@catenocrypt Hi..is there any example available for this in swift iOS??
